content/en/blog/2020/workload-entry/index.md

```diff
@@ -0,0 +1,100 @@
+
+The source of these complications is that Istio lacked a first class definition of a non-containerized workload, whose properties can be described independently of the service(s) it binds to.
+
+{{< image width="75%"
+    link="./workload-entry-first-example.svg"
+    alt="Service Entries Pointing to Workload Entries"
+    caption="The Internal of Service Entries Pointing to Workload Entries"
+    >}}
+
+### Workload Entry: A Non-Kubernetes Endpoint
+
+Workload Entry was created specifically to solve this problem. Workload Entry allows you to describe non-Pod endpoints that should still be part of the mesh, and treat them the same as a Pod. From here everything becomes easier, like enabling `MUTUAL_TLS` between workloads, whether they are containerized or not.
+
+To create a [Workload Entry](/docs/reference/config/networking/workload-entry/) and attach it to a [Service Entry](/docs/reference/config/networking/service-entry/) you can do something like this:
+
+{{< text yaml >}}
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: Workload Entry
+metadata:
+  name: vm1
+  namespace: ns1
+spec:
+  address: 1.1.1.1
+  labels:
+    app: foo
+    instance-id: vm-78ad2
+    class: vm
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: svc1
+  namespace: ns1
+spec:
+  hosts:
+  - svc1.internal.com
+  ports:
+  - number: 80
+    name: http
+    protocol: HTTP
+  resolution: STATIC
+  workloadSelector:
+    labels:
+      app: foo
+{{< /text >}}
+
+This creates a new Workload Entry with a set of labels and an address, and a Service Entry that uses a Workload Selector to select all endpoints with the desired labels, in this case including the Workload Entry that are  created for the VM.
+
+{{< image width="75%"
+    link="./workload-entry-final.svg"
+    alt="Service Entries Pointing to Workload Entries"
+    caption="The Internal of Service Entries Pointing to Workload Entries"
+    >}}
+
+Notice that the Service Entry can reference both Pods and Workload Entries, using the same selector. VMs and Pods can now be treated identically by Istio, rather than being kept separate.
+
+If you were to migrate some of your workloads to Kubernetes, and you choose to keep a substantial number of your VMs, the Workload Selector can select both Pods and VMs, and Istio will automatically load balance between them. The 1.6 changes also mean that Workload Selector syncs configurations between the Pods and VMs and removes the manual requirement to target both infrastructures with duplicate policies like mTLS and authorization.
+The Istio 1.6 release provides a great starting point for what will be possible for the future of Istio. The ability to describe what exists outside of the mesh the same way you do with a Pod leads to added benefits like improved bootstrapping experience. However, these benefits are merely side effects. For the first time, it requires _*no extra work*_ networking wise, to move services into the mesh.
```

Comment:
> Isn't it, at this point, inside the mesh?

A better word is Kube here, as that's the intention, since yes it is inside the mesh.

> Setting up all that config was extra work, no?
> (Can't really propose a change here sorry as I don't understand the aim of the paragraph)

The key point there is networking wise (arguably the hardest parts), before it would require each service to be aware of, and knowledgeable that there were two things, and know how to talk between the two. Yes there is still work to get the thing _inside_ the mesh, but now it is the same as a Pod/Service. Doesn't require everything to know there's some things "inside" Kube, and somethings "Outside" kube.

Reply:
You could go with:

"For the first time, it is the same amount of work to add services outside a Kubernetes environment as adding Pods and Services that are inside it."  (And then maybe something about bootstrapping)